src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -537,9 +545,35 @@ void GraphingCalculator::DisplayGraphSettings()
     flyoutGraphSettings->ShowAt(GraphSettingsButton);
 }
 
+void CalculatorApp::GraphingCalculator::AddTracePointerShadow()
+{
+    auto compositor = ::Hosting::ElementCompositionPreview::GetElementVisual(CursorPath)->Compositor;
+    auto dropShadow = compositor->CreateDropShadow();
+    dropShadow->BlurRadius = 6;
+    dropShadow->Opacity = 0.33f;
+    dropShadow->Offset = ::Numerics::float3(2, 2, 0); 
+    dropShadow->Mask = CursorPath->GetAlphaMask();
+
+    auto shadowSpriteVisual = compositor->CreateSpriteVisual();
+    shadowSpriteVisual->Size = ::Numerics::float2(static_cast<float>(CursorPath->ActualWidth), static_cast<float>(CursorPath->ActualHeight));
+    shadowSpriteVisual->Shadow = dropShadow;
+    ::Hosting::ElementCompositionPreview::SetElementChildVisual(CursorShadow, shadowSpriteVisual);
+}
+
 void GraphingCalculator::OnSettingsFlyout_Closing(FlyoutBase ^ sender, FlyoutBaseClosingEventArgs ^ args)
 {
     auto flyout = static_cast<Flyout ^>(sender);
     auto graphingSetting = static_cast<GraphingSettings ^>(flyout->Content);
     args->Cancel = graphingSetting->CanBeClose();
 }
+
+void GraphingCalculator::LeftGrid_SizeChanged(Object ^ /*sender*/, SizeChangedEventArgs ^ e)
+{
+    // Initialize the pointer to the correct location to match initial value in GraphControl\DirectX\RenderMain.cpp
+    TracePointer->Margin = Thickness(e->NewSize.Width/2 + 40, e->NewSize.Height/2 - 40, 0, 0);
```

Comment:
Why not set TracePointer.VerticalAlignment and TracePointer.HorizontalAlignment to Center and use the margin `40, -40, 0 0) to not have to manage that manually?

Reply:
We use the margin to reposition the cursor when arrow keys are pressed.  I think it's clearer to do the math here and have it mirror the math in RenderMain.cpp instead of having to recalculate when the point moves.  That said, I would really like to only have the math be done in one place (which is how it happens for everything except initialization), but I couldn't find a nice way to do that.

Reply:
Using a canvas to position the pointer now.